src/chrome/content/rules/Netfilter.org.xml

```diff
@@ -32,6 +32,7 @@
 	<target host="lists.netfilter.org" />
 	<target host="www.netfilter.org" />
 	<target host="workshop.netfilter.org" />
+	<target host="ipset.netfilter.org" />
```

Comment:
Please also remove the corresponding comment on the top and make sure that all checks are passing.